airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/task/DestinationTaskLauncher.kt

```diff
@@ -163,46 +159,43 @@ class DefaultDestinationTaskLauncher(
     }
 
     /** Called when a stream is ready for loading. */
-    override suspend fun handleStreamOpen(streamLoader: StreamLoader) {
-        log.info { "Registering stream open and loader available for ${streamLoader.stream}" }
-        streamLoaders[streamLoader.stream.descriptor]!!.complete(streamLoader)
+    override suspend fun handleStreamStarted(stream: DestinationStream) {
+        // Nothing to do because the SpillToDiskTask will trigger the next calls
```

Comment:
did this mean to say OpenStreamTask? (i.e. that's where we're invoking registerStartedStreamLoader now)

... do we even still need this method?

Reply:
First point: No, it's
```
start => SetupTask, SpillToDiskTask(s)
setupComplete => OpenStreamTask(s)
openComplete => this (do nothing)
each time spillToDisk finishes a file => ProcessRecordTask
```
And ProcessRecordTask awaits the stream loader to be available


Second point: It's philosophical? Should every task report its completion back to the task launcher, regardless of whether that causes anything to happen?